frontend/src/components/LocaleSwitcher.tsx

```diff
@@ -0,0 +1,41 @@
+import type { FallbackLng } from "i18next";
+import { useState } from "react";
+import {
+  Select,
+  SelectContent,
+  SelectItem,
+  SelectTrigger,
+  SelectValue,
+} from "src/components/ui/select";
+import { toast } from "src/components/ui/use-toast";
+import i18n, { supportedLocales } from "src/i18n/i18nConfig";
+
+export default function LocaleSwitcher() {
+  const fallbackLng = i18n.options.fallbackLng?.[0 as keyof FallbackLng];
```

Comment:
The fallback should not be optional, it's specified in the config

Reply:
Are you sure this is used correctly? it's not an array in the config

Reply:
Yeah this works, when I tried locally, it is also done the same way in LBE